### PR TITLE
Fix: bump db-to-cloud, show detailed LockError

### LINKED ISSUE #1268: Sync failed to acquire lock
* **Browser**: Firefox Nightly 91 (2021-06-28)
* **Operating System**: Arch Linux 5.12.13-lqx1-1-lqx
* **Stylus Version**: 1.5.17
* **Screenshot**:

image

Is there any way to reset the sync so it would work again? I'm not sure which browser is putting the locks in.

### PR DESCRIPTION
**Pull request by eight04**
Fixes #1268.

Please review `messages.json`

### CODE REVIEW

File: background/sync-manager.js [hunk 1]
```diff
@@ -208,6 +209,7 @@ const syncMan = (() => {
       setState(drive, state) {
         return chromeLocal.setValue(STORAGE_KEY + drive.name, state);
       },
+      retryMaxAttempts: 0,
```
Comment (eight04) [inline background/sync-manager.js:212]: @tophf Since there is already a retry mechanic in Stylus, I disabled this feature in db-to-cloud. Should use it?

**Comment by eight04** [comment]
image

File: background/sync-manager.js (hunk 1 above)
Comment (tophf) [reply background/sync-manager.js:212]: Now that db-in-cloud can retry, we should probably rely on it, and remove this code from Stylus.

File: _locales/en/messages.json [hunk 2]
```diff
@@ -1669,6 +1669,14 @@
     "message": "Sync failed.\nTry to re-login in Stylus options:\nclick 'disconnect' first, then 'connect'.",
     "description": "Tooltip for the toolbar icon"
   },
+  "syncErrorLock": {
+    "message": "The database is already in used. The lock will expire at $TIME$",
```
Comment (tophf) [inline _locales/en/messages.json:1673]: Remove `d` so it becomes `in use`.

**Comment by eight04** [comment]
image

**Review by tophf — APPROVED** [review]

**Pull request merged**